### PR TITLE
OSDOCS#7310: Adds notes for MS 4.13.8 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for MS 4.13.8 release

Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-7310

Link to docs preview:
https://63274--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-13-release-notes#microshift-4-13-8-dp

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
Looks good to me! I'm still new to reviewing, however, so please wait until @adellape approves of my review before proceeding. Thanks!

File: microshift_release_notes/microshift-4-13-release-notes.adoc
Comment (adellape) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:221]: I see that it's like this throughout the file (not just this PR), but should the word "image" be marked up here? Seems like the following would make sense:

```suggestion
For the `TopoLVM` image, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
```

If so, suggest changing throughout, either in this or a follow-up.

**Review by adellape — COMMENTED** [review]
@skopacz1 @opayne1 Just 1 additional question, but peer review comments LGTM 👍

File: microshift_release_notes/microshift-4-13-release-notes.adoc
Comment (opayne1) [reply microshift_release_notes/microshift-4-13-release-notes.adoc:221]: Oh yes, somehow along the way the image must have been pulled out of the backtick by accident. I will fix in this PR. Thanks for catching @adellape

**Pull request merged**
